### PR TITLE
Added ability to comment a projects

### PR DESCRIPTION
**Pull request by prodex**
issue #12

### CODE REVIEW

File: widgets/comment/Comment.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        if (!Yii::$app->user->isGuest) {
+            $commentForm = new CommentForm([
+                'scenario' => CommentForm::SCENARIO_CREATE,
+                'object_type' => $this->model->getObjectType(),
+                'object_id' => $this->model->getObjectId()
+            ]);
+            $commentForm->loadDefaultValues();
+
+            if ($commentForm->load(Yii::$app->request->post()) && $commentForm->save()) {
+                Yii::$app->getResponse()->refresh(sprintf("#c%d", $commentForm->id))->send();
```
Comment (samdark) [inline widgets/comment/Comment.php]: Could return here instead of using `Yii::$app->end()`.

Comment (prodex) [reply widgets/comment/Comment.php]: Now I looked, maybe `return` is not needed.

```php
Yii::$app->getResponse()->refresh(sprintf("#c%d", $commentForm->id))->send();
```

`send()` - itself closes the application.

PS: it's a widget, not a controller.

Comment (samdark) [reply widgets/comment/Comment.php]: `return` is to ensure no code from the rest of the method is executed.

Comment (prodex) [reply widgets/comment/Comment.php]: it is yes) Fixed.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request closed without merging**